src/minio.d.ts

```diff
@@ -104,7 +110,6 @@ export type LockUnit = RETENTION_VALIDITY_UNITS
 /**
  * @deprecated keep for backward compatible
  */
-export type LegalHoldStatus = LEGAL_HOLD_STATUS
```

Comment:
keep this maybe?

Reply:
Yes. This has been moved and re exported in this file.

Reply:
should also move deprecated comment then

Reply:
yes. thank you. i have removed the comment.